test/testdrive/status-history.td

```diff
@@ -9,189 +9,448 @@
+
+> CREATE CLUSTER sources SIZE '${arg.default-storage-size}', REPLICATION FACTOR 0
+
+> CREATE SOURCE auction
+  IN CLUSTER sources
+  FROM LOAD GENERATOR AUCTION FOR ALL TABLES
+
+> CREATE SOURCE kafka
+  IN CLUSTER sources
   FROM KAFKA CONNECTION kafka_conn (TOPIC 'testdrive-status-history-${testdrive.seed}')
-
-> CREATE TABLE kafka_source_tbl FROM SOURCE kafka_source (REFERENCE "testdrive-status-history-${testdrive.seed}")
+> CREATE TABLE kafka_tbl
+  FROM SOURCE kafka (REFERENCE "testdrive-status-history-${testdrive.seed}")
   FORMAT TEXT
-> COMMIT
 
-> CREATE CLUSTER kafka_sink_cluster SIZE '${arg.default-storage-size}';
-> CREATE SINK kafka_sink
-  IN CLUSTER kafka_sink_cluster
-  FROM kafka_source_tbl
+> CREATE SOURCE pg
+  IN CLUSTER sources
+  FROM POSTGRES CONNECTION pg_conn (PUBLICATION 'mz_source')
+> CREATE TABLE pg_tbl
+  FROM SOURCE pg (REFERENCE "pg_table")
+
+> CREATE SOURCE mysql
+  IN CLUSTER sources
+  FROM MYSQL CONNECTION mysql_conn
+> CREATE TABLE mysql_tbl
+  FROM SOURCE mysql (REFERENCE "public"."mysql_table")
+
+> SELECT o.name, s.status, s.details, s.replica_id
+  FROM mz_internal.mz_source_status_history s
+  JOIN mz_objects o ON o.id = s.source_id
+accounts       paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+auction        paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+auctions       paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+bids           paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+organizations  paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+users          paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+kafka          paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+kafka_tbl      paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+pg             paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+pg_tbl         paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+mysql          paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+mysql_tbl      paused    "{\"hints\":[\"There is currently no replica running this source\"]}"  <null>
+
+> SELECT o.name, s.status FROM mz_internal.mz_source_statuses s JOIN mz_objects o USING (id)
+accounts       paused
+auction        paused
+auctions       paused
+bids           paused
+organizations  paused
+users          paused
+kafka          paused
+kafka_tbl      paused
+pg             paused
+pg_tbl         paused
+mysql          paused
+mysql_tbl      paused
+auction_progress  running
+kafka_progress    running
+pg_progress       running
+mysql_progress    running
```

Comment:
These show up in `mz_source_statuses` and are always "running". Seems like another bug maybe?